src/base/bittorrent/private/filterparserthread.cpp

```diff
@@ -399,7 +399,7 @@ void FilterParserThread::processFilterList(libt::session *s, const QStringList &
     s->set_ip_filter(filter);
 }
 
-QString FilterParserThread::cleanupIPAddress(QString _ip)
+QString FilterParserThread::cleanup_IPv4Addr_LeadingZeros(QString _ip)
```

Comment:
Coding style is camelCase but it doesn't matter in this case because this method seems to be changed/removed in #5310.
If you could change it back to avoid rebasing #5310 that would be great.